client/examples/coffee-example/coffee-comparison-extension/src/browser/coffee-comparison-frontend-module.ts

```diff
@@ -13,7 +13,7 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
-import { ComparisonExtensionConfiguration } from 'comparison-extension/lib/browser/comparison-extension-configuration';
+import { ComparisonExtensionConfiguration } from '@eclipsesource/comparison-extension/lib/browser/comparison-extension-configuration';
```

Comment:
is the export wrong? why the weird import?

Reply:
I created a follow up issue for this: #21.